lib/node_modules/@stdlib/stats/base/dists/normal/ctor/benchmark/benchmark.js

```diff
@@ -71,6 +80,7 @@ bench( pkg+'::get:mu', function benchmark( b ) {
 			b.fail( 'should return set value' );
 		}
 	}
+
 	b.toc();
```

Comment:
```suggestion
	}
	b.toc();
```

Same comment as above, and applies throughout the PR

Reply:
This still needs to be addressed.